sklearn/preprocessing/data.py

```diff
@@ -397,6 +397,261 @@ def inverse_transform(self, X, copy=None):
         return X
 
 
+class RobustScaler(BaseEstimator, TransformerMixin):
+    """Standardize features by removing the median and scaling to IQR.
+
+    Centering and scaling happen independently on each feature (or each
+    sample, depending on the `axis` argument) by computing the relevant
+    statistics on the samples in the training set. Median and  interquartile
+    range are then stored to be used on later data using the `transform`
+    method.
+
+    Standardization of a dataset is a common requirement for many
+    machine learning estimators. Typically this is done by removing the mean
+    and scaling to unit variance. However, outliers can often influence the
+    sample mean / variance in a negative way. In such cases, the median and
+    the interquartile range often give better results.
+
+    Parameters
+    ----------
+    interquartile_scale: float or string in  ["normal" (default), ],
+           The interquartile range is divided by this factor. If
```

Comment:
Not sure if you're confused about the `interquartile_scale` parameter or picking the IQR as the range to scale in. In short, IQR is the most common range, yes.

To clear up the confusion about the `interquartile_scale`: it is used to make our estimate of the IQR into an (unbiased) estimate of the variance of the data.  For example, if the underlying data is a Normal distribution, we know that IQR = 1.349 * sigma^2.  The 1.349 is the `interquartile_scale`. So by dividing the IQR by that value and afterwards scaling the data by the resulting factor, we'll get data that has a variance ~ 1.  We could calculate the correct scale to use for any quantile range, not just the IQR.  However, for this to work we need to make assumptions about the underlying data distribution. 

I just tested it, and actually for this to work, the data must follow the assumed distribution pretty closely. To be honest, I think the `interquartile_range` should be removed, it just adds needless complexity.